packages/core-tests/models/resources/lib/defaultValue.test.ts

```diff
@@ -3,10 +3,11 @@ import { expect } from 'chai'
 import cf from 'clownface'
 import $rdf from 'rdf-ext'
 import { literal } from '@rdf-esm/data-model'
-import { xsd, rdf, foaf, dash } from '@tpluscode/rdf-ns-builders'
-import { NodeKindEnum } from '@rdfine/shacl'
+import { xsd, rdf, foaf, dash, sh } from '@tpluscode/rdf-ns-builders'
+import { NodeKind, NodeKindEnum } from '@rdfine/shacl'
 import { defaultValue } from '@hydrofoil/shaperone-core/models/resources/lib/defaultValue.js'
```

Comment:
The name of the file (not the function) is too general. 

I would call `defaultValue.js`  `URIMinting.js`, `CreateUri.js` or something like that, something that suggests what's its role

Reply:
Maybe but not quite. The function returns an initial value for a property. This can be an IRI, a blank node, or what is declared using `sh:defaultValue` property. That last predicate is obviously where the function got its name. Since the module only exports that on function, it is named thusly.

I can rename, no problem, but to something else.

Reply:
Ah! it's an ObjectValue :D